docs/examples/routes/httpd.yml

```diff
@@ -0,0 +1,13 @@
+name: httpd
+containers:
+- image: centos/httpd
```

Comment:
This image uses root,so it can't be run on default openShift configuration :(

Reply:
but you could use  `bitnami/nginx` for exaple

Reply:
fixed